public/app/features/teams/state/actions.ts

```diff
@@ -43,6 +44,20 @@ export function loadTeams(initial = false): ThunkResult<void> {
   };
 }
 
+export function loadUsersTeams(): ThunkResult<void> {
+  return async (dispatch) => {
+    // Early return if the user cannot list teams
+    if (!contextSrv.hasPermission(AccessControlAction.ActionTeamsRead)) {
+      let noTeams: Team[] = [];
+      dispatch(userTeamsLoaded(noTeams));
```

Comment:
```suggestion
      dispatch(userTeamsLoaded([]));
```
This code could be simplified.